src/main/java/nl/tudelft/contextproject/audio/BackgroundMusic.java

```diff
@@ -191,15 +178,11 @@ public synchronized AudioNode getCurrent() {
 	
 	/**
 	 * Called to indicate an update.
-	 * 
-	 * @param tpf
-	 * 		tick indicator
 	 */
-	public void update(double tpf) {
+	public void update() {
 		AudioNode an = current;
 		if (an == null) return;
-		
-		//Play the next song if the last one is done playing.
+
 		if (an.getStatus() == AudioSource.Status.Stopped) {
 			next();
 		}
```

Comment:
So you removed ALL my comments....
Well ok then....
I guess that means my code is pretty clear and understandable.